### PR TITLE
camerad: get embedded statistics and data by embedding in pixel data

### LINKED ISSUE #24230: camerad: Log camera temperatures
Should be possible to get in the frame readout extra data. Wonder if there's any correlation to camera malfunction.

The AR0231 appears to have two temp sensors, a "top" one and a "bottom" one.

### PR DESCRIPTION
**Pull request by pd0wm**
follow up of https://github.com/commaai/openpilot/pull/24299

```
0 - frame id: 1638
0 - temp 0 (top): 40.15
0 - temp 1 (bottom): 39.44
1 - frame id: 1638
1 - temp 0 (top): 40.30
1 - temp 1 (bottom): 40.15
2 - frame id: 1639
2 - temp 0 (top): 40.71
2 - temp 1 (bottom): 40.57

```

### CODE REVIEW

File: selfdrive/camerad/cameras/camera_common.cc [hunk 1]
```diff
@@ -147,6 +147,23 @@ bool CameraBuf::acquire() {
 
   double start_time = millis_since_boot();
 
+  if (camera_state->ci.registers_offset >= 0) {
```
Comment (pd0wm) [inline selfdrive/camerad/cameras/camera_common.cc:150]: Put somewhere that's called by  `processing_thread`.

**Comment by pd0wm** [comment]
```
( logMonoTime = 4458528715619,
  roadCameraState = (
    frameId = 429,
    encodeId = 0,
    timestampEof = 4458507904777,
    frameLength = 0,
    integLines = 221,
    globalGainDEPRECATED = 0,
    frameType = unknown,
    timestampSof = 4458476591641,
    transform = [1, 0, 0, 0, 1, 0, 0, 0, 1],
    lensPos = 0,
    lensSag = 0,
    lensErr = 0,
    lensTruePos = 0,
    gain = 0.8,
    recoverState = 0,
    highConversionGain = false,
    measuredGreyFraction = 0.23046875,
    targetGreyFraction = 0.22790544,
    processingTime = 0.019059086,
    temperaturesC = [41, 40.151516],
    frameIdSensor = 431 ),
  valid = true )
```

**Comment by pd0wm** [comment]
Good to merge after running in the testing closet for a bit.

**Comment by geohot** [comment]
This CI failure real? Can we merge this?

**Comment by adeebshihadeh** [comment]
I believe there were some issues found in the testing closet too.

**Pull request merged**